.changeset/brown-beers-tease.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firestore": minor
+"firebase-repo-scripts-prune-dts": minor
```

Comment:
This won't work, this package is ignored by changesets. Also I don't think there's any changes to it in this PR?

Reply:
Fixed. I modified some scripts at one point..